samples/bookinfo/build_push_update_images.sh

```diff
@@ -87,16 +95,19 @@ function run_vulnerability_scanning() {
   mkdir -p "$RESULT_DIR"
   # skip-dir added to prevent timeout of review images
   set +e
-  trivy image --ignore-unfixed --no-progress --exit-code 2 --skip-dirs /opt/ibm/wlp --output "$RESULT_DIR/$1_$VERSION.failed" "$2"
+  trivy image --ignore-unfixed --no-progress --exit-code 2 --skip-dirs /opt/ol/wlp --output "$RESULT_DIR/$1_$VERSION.failed" "$2"
   test $? -ne 0 || mv "$RESULT_DIR/$1_$VERSION.failed" "$RESULT_DIR/$1_$VERSION.passed"
   set -e
 }
 
 # Push images. Scan images if ENABLE_IMAGE_SCAN is true.
 for IMAGE in ${IMAGES};
 do
-  echo "Pushing: ${IMAGE}"
-  docker push "${IMAGE}";
+  # Multiarch images have already been pushed using buildx build	
+  if [[ "${ENABLE_MULTIARCH_IMAGES}" == "false" ]]; then	
+  	echo "Pushing: ${IMAGE}"
+  	docker push "${IMAGE}";
```

Comment:
nit: I wouldn't mind just making buildx always used, if it simplifies things

Reply:
so enable  the flag by default ? or just rm the flag and always build multi arch images ?

Reply:
we can have `--platforms` not always set but always use `docker buildx build`; I think its useful to have a flag to disable if it makes things simpler -- this script seems super complex for what it does (just build a few images), so I would definitely be in favor of simplifying it if we can

Reply:
sgtm, but would have to delete the code that checks if images are available locally or not since its not supported for multi arch images yet